### PR TITLE
fix: Object Oriented Programming: Use Dot Notation to Access the Properties of an Object

### PR DESCRIPTION
**Pull request by ezioda004**
Fixed typos.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/object-oriented-programming/use-dot-notation-to-access-the-properties-of-an-object.english.md [hunk 1]
```diff
@@ -21,10 +21,10 @@ Print both <code>properties</code> of the <code>dog</code> object below to your
 
 ```yml
 tests:
-  - text: Your should use <code>console.log</code> to print the value for the <code>name</code> property of the <code>dog</code> object.
-    testString: assert(/console.log\(.*dog\.name.*\)/g.test(code), 'Your should use <code>console.log</code> to print the value for the <code>name</code> property of the <code>dog</code> object.');
-  - text: Your should use <code>console.log</code> to print the value for the <code>numLegs</code> property of the <code>dog</code> object.
-    testString: assert(/console.log\(.*dog\.numLegs.*\)/g.test(code), 'Your should use <code>console.log</code> to print the value for the <code>numLegs</code> property of the <code>dog</code> object.');
+  - text: Your code should use <code>console.log</code> to print the value for the <code>name</code> property of the <code>dog</code> object.
```
Comment (RandellDawson) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/object-oriented-programming/use-dot-notation-to-access-the-properties-of-an-object.english.md]: I think the original intent was `Your` was supposed to be `You`, so there would be no need to add the word `code` here. 
```suggestion
  - text: You should use <code>console.log</code> to print the value for the <code>name</code> property of the <code>dog</code> object.
```

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/object-oriented-programming/use-dot-notation-to-access-the-properties-of-an-object.english.md [hunk 2]
```diff
@@ -21,10 +21,10 @@ Print both <code>properties</code> of the <code>dog</code> object below to your
...
+    testString: assert(/console.log\(.*dog\.name.*\)/g.test(code), 'Your code should use <code>console.log</code> to print the value for the <code>name</code> property of the <code>dog</code> object.');
```
Comment (RandellDawson) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/object-oriented-programming/use-dot-notation-to-access-the-properties-of-an-object.english.md]: No longer even need the message argument.
```suggestion
    testString: assert(/console.log\(.*dog\.name.*\)/g.test(code));
```

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/object-oriented-programming/use-dot-notation-to-access-the-properties-of-an-object.english.md [hunk 3]
```diff
@@ -21,10 +21,10 @@ Print both <code>properties</code> of the <code>dog</code> object below to your
...
+  - text: Your code should use <code>console.log</code> to print the value for the <code>numLegs</code> property of the <code>dog</code> object.
```
Comment (RandellDawson) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/object-oriented-programming/use-dot-notation-to-access-the-properties-of-an-object.english.md]: I think the original intent was `Your` was supposed to be `You`, so there would be no need to add the word `code` here.
```suggestion
  - text: You should use <code>console.log</code> to print the value for the <code>numLegs</code> property of the <code>dog</code> object.
```

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/object-oriented-programming/use-dot-notation-to-access-the-properties-of-an-object.english.md [hunk 4]
```diff
@@ -21,10 +21,10 @@ Print both <code>properties</code> of the <code>dog</code> object below to your
...
+    testString: assert(/console.log\(.*dog\.numLegs.*\)/g.test(code), 'Your code should use <code>console.log</code> to print the value for the <code>numLegs</code> property of the <code>dog</code> object.');
```
Comment (RandellDawson) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/object-oriented-programming/use-dot-notation-to-access-the-properties-of-an-object.english.md]: No longer even need the message argument.
```suggestion
    testString: assert(/console.log\(.*dog\.numLegs.*\)/g.test(code));
```

**Review by RandellDawson — DISMISSED** [review]
@ezioda004 Changes LGTM

**Review by RandellDawson — APPROVED** [review]

**Review by Manish-Giri — COMMENTED** [review]
The changes look good, however I'm thinking if we could get rid of the assert messages within `testString`. Those should already be covered by `text`.

That way if changes need to be made to the assert messages, it could only be done in one place, avoiding duplication like in this case.

**Comment by RandellDawson** [comment]
I will do that now.

**Review by Manish-Giri — APPROVED** [review]
LGTM. Thank You for your contribution.

**Pull request merged**